src/applications/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -2,6 +2,7 @@ import { connect } from 'react-redux';
 import React from 'react';
 import moment from 'moment';
 import recordEvent from '../../../platform/monitoring/record-event';
+import localStorage from '../../../platform/utilities/storage/localStorage';
 
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
```

Comment:
In most cases, we're accessing localStorage without the formal `window` prefix, so we can just override the local variable `localStorage` in a top-level import.

Reply:
We might want to add a ticket to look into enforcing this with eslint. It's going to be easy to forget to import this module instead of using the global. But I think we could either write a custom rule or remove localStorage from the allowed globals list in eslint.

Reply:
Great idea, moved to this ticket - department-of-veterans-affairs/vets.gov-team/issues/11283